warp10/src/main/java/io/warp10/script/ext/sensision/SensisionWarpScriptExtension.java

```diff
@@ -19,29 +19,41 @@
 import java.util.HashMap;
 import java.util.Map;
 
+import io.warp10.WarpConfig;
 import io.warp10.warp.sdk.WarpScriptExtension;
 
 /**
  * WarpScript Extension which exposes functions to
  * set/update/clear/get Sensision metrics
  */
 public class SensisionWarpScriptExtension extends WarpScriptExtension {
-  
+
   private static final Map<String, Object> functions;
-  
+
+  public static final String READ_CAPABILITY = "sensision.read";
+  public static final String WRITE_CAPABILITY = "sensision.write";
+  private static final String CONF_USE_CAPABILITY = "sensision.capability";
```

Comment:
Please add the 'sensision.capability' configuration key to configuration template files.